packages/expo-linear-gradient/src/LinearGradient.tsx

```diff
@@ -78,7 +78,7 @@ export type LinearGradientProps = ViewProps & {
 export class LinearGradient extends React.Component<LinearGradientProps> {
   render() {
     const { colors, locations, start, end, dither, ...props } = this.props;
-    let resolvedLocations = locations;
+    let resolvedLocations = locations as number[] | undefined;
```

Comment:
Nit, prefer typing over casting. In this case we miss `undefined` in there.

```suggestion
    let resolvedLocations: number[] | null | undefined = locations;
```

or

```suggestion
    let resolvedLocations: LinearGradientProps['locations']  = locations;
```

Reply:
Thanks, had to apply an extra `readonly` for it to typecheck.